frontend/src/types/Chat.ts

```diff
@@ -17,13 +17,21 @@ export interface UserChatPermissions {
 export interface SingleMessage {
 	id: number;
 
-	sender: User;
-	chat: Chat;
+	sender?: User;
```

Comment:
sender would be the full relationship object of the user. There is a `user_id` property that is used for when this object is used for communicating through sockets.

the sender property is mainly there as a fallback for when user_id is not set while rendering the message component (actually made a change there because of your comment here :P)